justfile

```diff
@@ -41,7 +41,11 @@ test-wkspc:
     cargo test --workspace -- --nocapture
 
 # Execute our python integration tests inside /tests using nix for all setup needed.
-test-int-nix:
+test-int-nixrun:
```

Comment:
Instead of having the development shell run and exit, can we just define a run command here like `nix develop .#pythonTests -c "poetry run poe tests"`. That way we could hop into the pythonTests shell for debugging too. Maybe just me, but I kinda like keeping the dev shells interactive (vs. a hard coded exit)/